I/ITSOL_2/build_tarballs.jl

```diff
@@ -0,0 +1,68 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ITSOL_2"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://www-users.cs.umn.edu/~saad/software/ITSOL/ITSOL_2.tar.gz",
```

Comment:
It's a bit concerning these URLs are unversioned, I hope they won't ever be replaced... Well, unless there are alternatives I guess we don't have many choices.

Reply:
Well, the code has not been updated in years, maybe bigger concern is how long it will available.
I suppose I could import the library into GitHub, would be more likely be more reliable too.